column/__init__.py

```diff
@@ -7,10 +7,16 @@
 from subprocess_runner import SubprocessRunner
 from callback import AnsibleCallback
 
+import ConfigParser
+
+DEFAULT_CONF_FILE = '/etc/column/column.conf'
 
 __all__ = [
     'APIRunner', 'Runner', 'SubprocessRunner', 'AnsibleCallback'
 ]
 
 
 __version__ = '0.3.6'
+
+cfg = ConfigParser.ConfigParser(allow_no_value=True)
+cfg.read(DEFAULT_CONF_FILE)
```

Comment:
according to the doc, https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.read

If none of the named files exist, the ConfigParser instance will contain an empty dataset.
So i think it is okay now